error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/NestedOptionalsTest.java

```diff
@@ -39,4 +39,19 @@ void identification() {
             "}")
         .doTest();
   }
+
+  @Test
+  void failingTypeCheck() {
```

Comment:
Slightly more in line which what we do in a few other places
```suggestion
  void identificationOptionalTypeNotLoaded() {
```

(Funny: I tried to replace the `Duration` expression below with `Stream.empty()`, but then the NPE didn't happen. I think because `Stream` does reference `Optional`, while `Duration` (and `ChronoUnit`) don't.